config/crd/bases/flows.netobserv.io_flowcollectors.yaml

```diff
@@ -2136,6 +2136,2129 @@ spec:
             type: object
         type: object
     served: true
+    storage: false
```

Comment:
The conversion webhook configuration lives within the CRD object itself, I don't see the configuration here so perhaps this is why the conversion isn't working presently

Reply:
its under crd/bases/patches/webhook_in_flowcollectors.yaml